packages/flat-config-disable-type-checked/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "module": "NodeNext",
     "moduleResolution": "NodeNext",
+    "noEmit": true,
```

Comment:
This should have been on before, so that if you ran `npm run tsc` before `npm run lint` you wouldn't have extra `src/*.js` files to lint.